kubernetes/template/DeploymentTemplates/script.sh

```diff
@@ -147,6 +147,12 @@ convert_secret_to_cert()
     PASSWORD=$(cat cert.json | jq '.password' | tr -d \")
     
     openssl pkcs12 -in $2 -nodes -passin pass:$PASSWORD -out $3
+    
+    log_level -i "Converting to certificate"
+    openssl pkcs12 -in $2 -clcerts -nokeys -out $4 -passin pass:$PASSWORD
```

Comment:
Could you please update the function documentation to include a description for $4 and $5?